### ISSUE TITLE
events calendar portlet uses <h3> for "add ... to calendar" 1

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#366]
This breaks page hierarchy. Should be ```<h2>``` but look and feel should not change.

Found on ```/events``` and on each individual event page.

**Comment by cguardia** [comment]
This is on staging.

**Comment by paregorios** [comment]
This looks good. Please merge to master and deploy to production.
